src/index.html

```diff
@@ -11,7 +11,126 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div>
+        <fieldset class="section">
+          <legend>Personal information</legend>
+          <label>
+            Surname:
+            <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+            >
```

Comment:
```suggestion
            <input
              type="text"
              name="surname"
              autocomplete="off"
            >
```

fix indentation in the whole document
(checklist points 2 and 3)